### PR TITLE
Add new package: py-boom

### PR DESCRIPTION
**Pull request by darmac**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-boom/package.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyBoom(PythonPackage):
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-boom/package.py]: I'm a little concerned about the package name. There is already a `boom` package on PyPI that is not the same as this one. Do they have a PyPI release?

Comment (darmac) [reply var/spack/repos/builtin/packages/py-boom/package.py]: I found that:
https://pypi.org/project/boom2/

boom2 is just a `hello world` project and have no description on `PyPI` website, 
It looks no update for that PyPI package.

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-boom/package.py]: If they haven't released on PyPI, we can give them any name that isn't already taken on PyPI, like `py-boom-boot` or `py-boot-manager`

Comment (darmac) [reply var/spack/repos/builtin/packages/py-boom/package.py]: So how about `py-boom-boot-manager`, that may descript the `py-boom` feature and reduce conflict rate on `PyPI`.

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-boom/package.py]: That's fine with me. If the devs don't like the name, they can always apply for a PyPI account.

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**